### PR TITLE
Allow using phpdbg SAPI

### PR DESCRIPTION
**Pull request by danog**

### CODE REVIEW

File: lib/Context/Process.php [hunk 1]
```diff
@@ -80,9 +80,13 @@ public function __construct($script, string $cwd = null, array $env = [], string
             "log_errors" => "1",
         ];
 
+        $otherOpts = [];
         if ($binary === null) {
             if (\PHP_SAPI === "cli") {
                 $binary = \PHP_BINARY;
+            } else if (\PHP_SAPI === "phpdbg") {
+                $binary = \PHP_BINARY;
+                $otherOpts []= '-qrr';
```
Comment (kelunik) [inline lib/Context/Process.php:89]: I'm not sure whether it makes sense to start the sub-processes with `phpdbg`. Shouldn't this just fall back to the located CLI binary?

Comment (danog) [reply lib/Context/Process.php:89]: It is often useful, for example if you need to gather coverage info without installing specific extensions (which, by the way, are way more resource-heavy than phpdbg's coverage driver).

Comment (kelunik) [reply lib/Context/Process.php:89]: Generally, the coverage collection won't be enabled in the sub-process, no?

Comment (danog) [reply lib/Context/Process.php:89]: But with this MR, you __can__ enable it if needed, for example if using directly https://github.com/sebastianbergmann/php-code-coverage, like in my case.

**Review by kelunik — APPROVED** [review]

**Pull request merged**